CONTRIBUTING.md

```diff
@@ -167,7 +167,7 @@ These include:
 * All classes should have `__repr__` and `__str__` functions
 * Docstrings use `Note` instead of `Notes`
 * Try to avoid creating a try/except statement where except passes
-* Use setup and teardown in test classes
+* Use setup_method (or setup_class) and teardown_method (or teardown_class) in test classes
```

Comment:
We have shifted to using specific method and class level setup and teardown, since pytest is dropping support for the old style.